tutor/src/components/onboarding/payment-options.tsx

```diff
@@ -0,0 +1,37 @@
+import { React, observer } from 'vendor'
+import { Button } from 'react-bootstrap'
+import { OnboardingNag, Body } from './onboarding-nag'
+
+import type { StudentCourseOnboarding } from './ux/student-course'
+
+interface PaymentOptionsProps {
+    ux: StudentCourseOnboarding
+}
+
+@observer
+export default class PaymentOptions extends React.Component<PaymentOptionsProps> {
+    static className = 'payment-options';
```

Comment:
is className used? It's hard to search for but I didn't see it anywhere

Reply:
Oh good catch, removing that.